projects/plugins/launchpad/src/js/components/Admin.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { AdminPage, AdminSection, AdminSectionHero } from '@automattic/jetpack-components';
+
+/**
+ * Internal dependencies
+ */
+import useConnection from '../hooks/useConnection';
+import './admin-style.scss';
+
+/* eslint react/react-in-jsx-scope: 0 */
+const Admin = () => {
+	const [ connectionStatus, renderConnectScreen, renderConnectionStatusCard ] = useConnection();
+	const [ connectionLoaded, setConnectionLoaded ] = useState( false );
+
+	useEffect( () => {
+		if ( 0 < Object.keys( connectionStatus ).length ) {
+			setConnectionLoaded( true );
+		}
+	}, [ connectionStatus ] );
+
+	const isFullyConnected = () => connectionLoaded && connectionStatus.isUserConnected && connectionStatus.isRegistered;
```

Comment:
There's no need to make this a function. If the `connectionLoaded` changes, React rerender the component and `const` value will be updated.

```suggestion
        const isFullyConnected = connectionLoaded && connectionStatus.isUserConnected && connectionStatus.isRegistered;
```

Reply:
Whoops. Fixed in 684199c